index.html

```diff
@@ -0,0 +1,723 @@
+        <a href="#"><i class="fas fa-search fa-sm"></i></a>
+        <a href="#space&cosmos">SPACE & COSMOS</a>
+      </div>
+
+      <a href="#"
+        ><img
+          src="http://assets.stickpng.com/images/592fe3133919fe0ee3614d93.png"
+          alt="New York times"
+          class="newyork-logo"
+      /></a>
+
+      <div class="nav-right">
+        <button type="submit" class="btn info">SUBSCRIBE NOW</button>
+        <button type="submit" class="btn info">LOG IN</button>
+      </div>
+    </div>
+
+    <!--TOP HEADING-->
+    <div class="main">
+      <!--TOP IMAGE-->
+      <div class="image">
+        <div class="image-heading">
+          <h1>Space Ripples Reveal Big Bang's <br />Smoking Gun</h1>
+        </div>
+        <div class="social-icons">
+          <a href="#"><i class="fab fa-facebook-f"></i></a>
+
+          <a href="#"><i class="fab fa-whatsapp"></i></a>
+
+          <a href="#"><i class="fab fa-twitter"></i></a>
+
+          <a href="#"><i class="fas fa-envelope fa-sm"></i></a>
+
+          <a href="#"><i class="fas fa-share fa-sm"></i></a>
+
+          <a href="#"><i class="far fa-bookmark"></i></a>
+        </div>
+
+        <img
+          src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-superJumbo.jpg?quality=90&auto=webp"
+          width="1000"
+          height="500"
+          alt="BIG IMAGE HERE"
+        />
+        <div class="figcaption">
+          Alan Guth was one of the first physicists to hypothesize the existence
+          of inflation, which explains how the universe expanded so uniformly
+          and so quickly in the instant after the Big Bang 13.8 billion years
+          ago.<cite>Rick Friedman for The New York Times</cite>
+        </div>
+      </div>
+
+      <!--FIRST PARAGRAPH SECTION MIDDLE-->
+      <div class="section-with-aside">
+        <aside>
+          <div class="editor-picks">Editors’ Picks</div>
+          <ul class="no-bullets">
+            <li>
+              <div class="editor-picks-card">
+                <img
+                  src="https://github.com/Stephanie041996/Newyork/blob/pagefeatures/images/16Windolf-square640-v2.jpg"
+                  alt="placeholder1"
+                  width="100"
+                  height="100"
+                />
+                <p>
+                  Aut culpa nam officia natus, deleniti illo quisquam, omnis
+                  ratione obcaecati!
+                </p>
+              </div>
+            </li>
+            <li>
+              <div class="editor-picks-card">
+                <img
+                  src="https://github.com/Stephanie041996/Newyork/blob/pagefeatures/images/16judas-notebook1-threeByTwoSmallAt2X-v3.jpg"
+                  alt="placeholder2"
+                  width="100"
+                  height="100"
+                />
+                <p>
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut,
```

Comment:
## Improve styling on the `editor's pick` section
- [ ] Great job having implemented the `editor's pick` section using `aside` but it is wrongly placed. The images are not loading either. Kindly fix this.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://stephanie041996.github.io/Newyork/)|
|:--:|:--:|
|![editor's-pick](https://user-images.githubusercontent.com/22345961/108399372-15cf4f00-722b-11eb-920d-a2bb53eb20e9.png)|![editor's-pick-clone](https://user-images.githubusercontent.com/22345961/108399385-1a940300-722b-11eb-8ee7-cc67656c28b8.png)|